wlts/json_configs/feature_collection.json

```diff
@@ -7,6 +7,7 @@
       "detail": "http://www.obt.inpe.br/OBT/assuntos/programas/amazonia/deter",
       "datasource_id": "3c20cbb4-ca94-4c1f-99af-6377f30bc683",
       "dataset_type": "Feature",
+      "is_public": "True",
```

Comment:
You may consider to use bool type [json.org](https://www.json.org/json-en.html#text)

Reply:
Done, thank you!